roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -290,6 +290,24 @@
       path: "{{ generated_certs_dir }}/passwd.yml"
 
 # services
+# bz1636248 - need to delete the service if it exists because clusterIP is immutable
+- name: Check to see if logging-{{ es_component }}-cluster service
+  oc_obj:
+    state: list
+    kind: service
+    name: "logging-{{ es_component }}-cluster"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+  register: logging_elasticsearch_cluster_service
+
+- name: Remove logging-{{ es_component }}-cluster service
+  oc_service:
+    state: absent
+    name: "logging-{{ es_component }}-cluster"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+  when:
+  - logging_elasticsearch_cluster_service.results.returncode == 0
```

Comment:
is this still 0 even if the object doesn't yet exist?

Reply:
its 1 if it doesnt exist